shader/backdrop_dyn.wgsl

```diff
@@ -34,6 +46,9 @@ fn main(
         sh_row_width[local_id.x] = path.bbox.z - path.bbox.x;
         row_count = path.bbox.w - path.bbox.y;
         sh_offset[local_id.x] = path.tiles;
+    } else {
+        // Explicitly zero the row width, just in case.
```

Comment:
It isn't very clear to me what "just in case" here means.
I see that we're not doing a scan over `sh_row_width`, and so in theory we won't ever be reading this value.

To be clear, I think this change is fine - especially once we start to use gfx-rs/wgpu#5508.
But is this actually fixing an issue, or just programming defensively?

Reply:
My analysis matches Daniel's. In any case, this code won't be around very long, I want to replace it with partition-wide prefix sum of the backdrop values, so being defensive seems preferable to over-optimizing.

Reply:
This is indeed being defensive for the case where workgroup memory initialization might be turned off. The scan in the following lines is over `sh_row_count` but I think it's technically possible for the loop at the very bottom to read this `sh_row_width` value if `el_ix` happens to match `local_id.x`?

At any rate, I think I'll leave this in and land this as is.